front/src/common/redux/students/types.js

```diff
@@ -0,0 +1,4 @@
+export const REGISTER_STUDENT = 'REGISTER_STUDENT';
+export const STUDENT_COMPLETED_REG = 'STUDENT_COMPLETED_REGISTRATION';
+export const REG_STUDENT_FAILED = 'REGISTRATION_STUDENT_FAILED';
+export const STUDENT_REG_LOGICAL_ERROR_HAPPENED = 'STUDENT_REGISTRATION_LOGICAL_ERROR_HAPPENED';
```

Comment:
Hey, I meant another thing related to names. `REG` sounds weird, please use `REGISTER` as a shorter possible name. 
I meant the word `LOGICAL` sounds quite strange. Do we really need it?

Reply:
Changed naming.